bom/openhab-core/pom.xml

```diff
@@ -514,6 +514,12 @@
       <version>${project.version}</version>
       <scope>compile</scope>
     </dependency>
+    <dependency>
```

Comment:
Is this still needed?

Reply:
AFAIK the BOMs are required for generating the indices used by Bnd for resolving the "runbundles" in .bndrun files (itests/Eclipse).

Reply:
For me it is sufficient to add the bundle to the [demo-app POM](https://github.com/openhab/openhab-distro/blob/main/launch/app/pom.xml), at least the resolver fails if I remove the bundle from the BOM and succeeds after I add it to the POM.

Reply:
Perhaps the index also improves performance?

Besides that the BOMs are used for managing versions and to make sure all bundles are build before feature validation starts.